recipes/pantools/build.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
+
 set -eu -o pipefail
 
-outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
-mkdir -p $outdir
+PACKAGE_HOME=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
```

Comment:
It's not really important, but `/opt/pkg-1.2.3` is usually for things that bring their own `/bin`/`lib`/etc structure and are isolated from the system. `/share/pkg/` is just fine for noarch data such as jars. `/lib/pkg/` would be as well, actually, it's where architecture dependent stuff would go.


YI, but IMO `share/PKG_NAME/` is just fine,